tools/migrate_photo_from_odp.rb

```diff
@@ -0,0 +1,17 @@
+# Migrate all photos with restricted license (0) from the Open bucket to the Static bucket
+
+max_photo_id = LocalPhoto.last.id
+batch_size = 100000
+index = 0
+
+while index < max_photo_id
+	LocalPhoto.where( license: 0, file_prefix_id: 1 ).
+	where("id between ? and ?", index, index + batch_size -1).
+	find_each do |photo|
```

Comment:
minor detail here. `find_each` will by default fetch up to 1000 records sorted by ID, and if there are more than 1000 records it will make additional queries where ID > the last ID, again sorted by ID and limited to 1000. Sometimes that's helpful, but in this case the batches are being handled in a custom way because we know the table is huge. I would prefer to see this as an `each` and not a `find_each` to save the sort and limit in the query in hopes it would be a little more efficient